app/src/main/java/com/android/shelfLife/ui/camera/Scanner.kt

```diff
@@ -133,11 +133,14 @@ fun BarcodeScannerScreen(
                       coroutineScope.launch { sheetScaffoldState.bottomSheetState.hide() }
                       Log.d("BarcodeScanner", "Cancelled")
                     },
-                    foodItemViewModel = foodItemViewModel,
-                )
+                    onExpandRequested = {
+                        // When the partially expanded content is clicked, expand the sheet fully
+                        coroutineScope.launch { sheetScaffoldState.bottomSheetState.expand() }
+                    })
               }
               Spacer(modifier = Modifier.height(100.dp))
             },
+            sheetPeekHeight = 240.dp,
```

Comment:
> - nit picky: hardcoded height might be too big on small phones.
> - there are also other hardcoded sizes or paddings on the screen, but I think it is fine since the focus of this PR is not on aesthetics but rather functionality